### ISSUE TITLE
update readme?

### ISSUE THREAD

**Issue by westwick** [sphilee/jsPDF-CustomFonts-support#55]
Hey just want to confirm... I think the main jspdf lib incorporated custom font support directly without the need for this package any longer, as of 1.5. Unless I'm missing something. Is that correct? I removed this package after updating and everything seems to be working. I guess this package remains just to support projects still using jspdf 1.4 and under. Maybe the documentation should be updated? Would be happy to do so if the above is correct

**Comment by sphilee** [comment]
You are right. This repository seems to need to be deprecated. You can use jsPDF v.1.5.0 or later.
